src/hooks/use-alert.ts

```diff
@@ -41,8 +53,38 @@ export function useAlert() {
     [toast]
   );
 
+  const success = useCallback(
+    ({ id, title, message }: AlertArguments) =>
+      toast({
+        id,
+        title,
+        description: trimMessage(message),
+        status: "success",
+        position: "top",
+        isClosable: true,
+        duration: 1000,
+      }),
+    [toast]
+  );
+
+  const warning = useCallback(
+    ({ id, title, message }: AlertArguments) =>
+      toast({
+        id,
+        title,
+        description: trimMessage(message),
+        status: "warning",
+        position: "top",
+        isClosable: true,
+        duration: 1000,
```

Comment:
Can we make warning a little bit long?
```suggestion
        duration: 3000,
```

Reply:
I have extended the warning messages to 3000 and the success message to 2000.